app/main.py

```diff
@@ -1,8 +1,43 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
```

Comment:
The `people` dictionary assumes that all names are unique. If there are duplicate names, this will overwrite previous entries. Consider using a unique identifier or handling duplicates appropriately.